__tests__/app.test.js

```diff
@@ -73,18 +74,48 @@ describe('GET/api/articles/:article_id', () => {
     it('responds with status 404 if article_id is not found', () => {
         const articleId = 999999;
         return request(app)
-        .get(`/api/articles/${articleId}`)
-        .then((res) => {
+          .get(`/api/articles/${articleId}`)
+          .expect(404)
+          .then((res) => {
             const { body } = res;
-            const { msg } = body;
-            expect(msg).toBe(`Article ${articleId} not found`);
+            expect(body.msg).toBe(`Article ${articleId} not found`);
         });
     });
     it('responds with status 400 if article_id is not a number', () => {
-        const articleId = 'not-a-number';
+        const articleId = 'not_a_number';
         return request(app)
-        .get(`/api/articles/${articleId}`)
-        .expect(400)
-        .send({msg: "Invalid article ID"});
+          .get(`/api/articles/${articleId}`)
+          .expect(400)
+          .then((res) => {
+            const { body } = res;
+            const { msg } = body;
+            expect(msg).toBe("Invalid article ID");
+        });
     });
 });
+
+describe("GET /api/articles", () => {
```

Comment:
Need to add tests for error handling,
What if the article_id given is not a number?
What if it is a number but it doesn't exist?